packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
```

Comment:
`fireEvent` still works but is deprecated.  You'd be better off using `userEvent.setup()` from [`@testing-library/user-event@14`](https://testing-library.com/docs/user-event/intro).